source/GameInterface/Services/Settlements/Messages/SettlementChangedBribePaid.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Common.Messaging;
+
+namespace GameInterface.Services.Settlements.Messages;
+
+/// <summary>
+/// When the server changes BribePaid
+/// </summary>
+public record SettlementChangedBribePaid : ICommand
```

Comment:
This should be an IEvent

Reply:
Incorrect, This goes from GI->Server->Client->GI. Changed comment so it makes more sense.